spec/rubocop/cop/rspec/multiple_describes_spec.rb

```diff
@@ -3,28 +3,36 @@
 RSpec.describe RuboCop::Cop::RSpec::MultipleDescribes do
   subject(:cop) { described_class.new }
 
-  it 'finds multiple top level describes with class and method' do
+  it 'flags multiple top-level example groups with class and method' do
     expect_offense(<<-RUBY)
       describe MyClass, '.do_something' do; end
-      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Do not use multiple top level describes - try to nest them.
+      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Do not use multiple top-level example groups - try to nest them.
       describe MyClass, '.do_something_else' do; end
     RUBY
   end
 
-  it 'finds multiple top level describes only with class' do
+  it 'flags multiple top-level example groups only with class' do
     expect_offense(<<-RUBY)
       describe MyClass do; end
-      ^^^^^^^^^^^^^^^^ Do not use multiple top level describes - try to nest them.
+      ^^^^^^^^^^^^^^^^ Do not use multiple top-level example groups - try to nest them.
       describe MyOtherClass do; end
     RUBY
   end
 
-  it 'skips single top level describe' do
+  it 'ignores single top-level example group' do
     expect_no_offenses(<<-RUBY)
-      require 'spec_helper'
-
       describe MyClass do
       end
     RUBY
   end
+
+  it 'flags shared example groups' do
+    expect_offense(<<-RUBY)
+      shared_examples_for 'behaves' do
+      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Do not use multiple top-level example groups - try to nest them.
```

Comment:
This is a bigger change, but I believe a good one.

Reply:
I think it's quite common to define multiple shared examples in a single, helper file?

Reply:
Agree, will narrow it back down to example groups only 👍

We may think of this option later on.